app/controllers/paginable/departments_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Paginable::DepartmentsController < ApplicationController
+
+  include Paginable
+
+  # /paginable/Departments/index/:page
+  def index
+
+    paginable_renderise(
+      partial: "index",
+      scope: Departments.by_org(current_user.org),
```

Comment:
I'm unfamiliar with the `.by_org` built-in, is this provided by the Paginable package?
I would have expected this to be `current_user.org.departments`